File: EUREKA_BANK_RESTFUL_DOTNET/EUREKA_BANK_CLIMOV_RESTFUL2/app/src/main/java/ec/edu/eureka_bank_climov_restful/model/Movimiento.java
package ec.edu.eureka_bank_climov_restful.model;

public class Movimiento {
    private String fecha;
    private String tipo;
    private String accion;
    private double importe;

    public String getFecha() {
        return fecha;
    }

    public String getTipo() {
        return tipo;
    }

    public String getAccion() {
        return accion;
    }

    public double getImporte() {
        return importe;
    }
}

File: EUREKA_BANK_RESTFUL_DOTNETO_NUBE/EUREKA_BANK_Servidor_SOAP/EUREKA_BANK_Servidor_RESTFUL/Models/Movimiento.cs
using System.ComponentModel.DataAnnotations.Schema;

namespace EUREKA_BANK_Servidor_RESTFUL.Models
{
    [Table("Movimiento")]
    public class Movimiento
    {
        [Column("chr_cuencodigo")]
        public string CuentaId { get; set; }

        [Column("int_movinumero")]
        public int NumeroMovimiento { get; set; }

        [Column("dtt_movifecha")]
        public DateTime Fecha { get; set; }

        [Column("chr_tipocodigo")]
        public string Tipo { get; set; } // Aquí aún es el código tipo, no descripción

        [NotMapped]
        public string Accion => new[] { "001", "003", "005", "008" }.Contains(Tipo?.PadLeft(3, '0')) ? "INGRESO" : "SALIDA";

        [Column("dec_moviimporte")]
        public decimal Importe { get; set; }

        [Column("chr_cuenreferencia")]
        public string? CuentaDestino { get; set; }

        [Column("chr_emplcodigo")]
        public string EmpleadoId { get; set; }
    }
}
